purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -138,6 +139,29 @@ internal class DiagnosticsTracker(
         }
     }
 
+    fun trackFeatureNotSupported(
+        playStorePackageInfo: PackageInfo?,
+        playServicesPackageInfo: PackageInfo?,
+        billingResponseCode: Int,
+        billingDebugMessage: String,
+    ) {
+        val event = DiagnosticsEntry.Counter(
+            name = DiagnosticsCounterName.FEATURE_NOT_SUPPORTED_ERROR,
+            tags = mapOf(
+                "play_store_version" to (playStorePackageInfo?.versionName ?: ""),
+                "play_store_first_install_time" to (playStorePackageInfo?.firstInstallTime ?: "").toString(),
+                "play_store_last_update_time" to (playStorePackageInfo?.lastUpdateTime ?: "").toString(),
+                "play_services_version" to (playServicesPackageInfo?.versionName ?: ""),
+                "play_services_first_install_time" to (playServicesPackageInfo?.firstInstallTime ?: "").toString(),
+                "play_services_last_update_time" to (playServicesPackageInfo?.lastUpdateTime ?: "").toString(),
+                "billing_response_code" to billingResponseCode.toString(),
+                "billing_debug_message" to billingDebugMessage,
```

Comment:
We decided to store this as a Counter since we're interested in the total number of events, but since we're able to gather more granular into about the package versions, would it make sense to store this as an Event? This might be too many tags with unique values for a counter.

Reply:
Will remove timestamps as they are not indispensable, and leave package versions only.